task/sendmail/0.2/sendmail.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sendmail
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Messaging
+    tekton.dev/tags: mail
+    tekton.dev/displayName: "send mail"
+    tekton.dev/platforms: "linux/amd64,linux/s390x,linux/ppc64le,linux/arm64"
+spec:
+  description: >-
+    This task sends a simple email to receivers via SMTP server
+
+  params:
+  - name: server
+    type: string
+    description: secret name for SMTP server information (url, port, password)
+  - name: subject
+    type: string
+    description: plain text email subject
+  - name: body
+    type: string
+    description: plain text email body
+  - name: sender
+    type: string
+    description: sender email address
+  - name: recipients
+    type: string
+    description: recipient email addresses (space delimited list)
+  steps:
+  - name: send
+    image: docker.io/library/python:3.8-alpine@sha256:e11bbd37d4371894e954421b85dbe8dd4eb7198d7cb4ed144ab529f19f57c3f1 #tag: 3.8-alpine
```

Comment:
should we consider bumping up the python image to something latest? Current one might have CVEs. Also python 3.12 is out so we can probably use that. Please let me know your thoughts

Reply:
I have updated the image version 3.12.